### PR TITLE
Crouching animation

### PR DESCRIPTION
**Pull request by kbatbouta**
## Addition 

- Crouching animations.
- Visual feedback aka: crouching animations.
- broken neck simulator.

### CODE REVIEW

**Comment by AluminumAlman** [comment]
Screenshots of the crouching animation (provided by Andross):
image
image
image
image

I don't really like how much the heads are detached from the body, maybe the offset should be reduced a bit, and maybe implement some leaning on the body from the sides, scaling from the front/back (to emulate leaning)?

**Comment by kbatbouta** [comment]
Scaling is not feasible at all.

**Comment by kbatbouta** [comment]
> Screenshots of the crouching animation (provided by Andross):
> image
> image
> image
> image
> 
> I don't really like how much the heads are detached from the body, maybe the offset should be reduced a bit, and maybe implement some leaning on the body from the sides, scaling from the front/back (to emulate leaning)?

These are taken under different offset levels. I suggest trying it.

**Comment by AluminumAlman** [comment]
What do you think about such offsets:
image

image
(normal)

image
(crouched, the head's height does not change depending on the crouch direction)

Also, would it be a good addition to have suppressed pawns have the same offset to the head as well?

File: Languages/English/Keyed/ModMenu.xml [hunk 1]
```diff
@@ -13,6 +13,9 @@
 
   <!-- ========== General settings ========== -->
 
+  <CE_Settings_CrouchingAnimation_Title>Enabel crouching animations</CE_Settings_CrouchingAnimation_Title>
```
Comment (Alicecomma) [inline Languages/English/Keyed/ModMenu.xml:16]: Enable

**Pull request closed without merging**